DQM/SiPixelPhase1Summary/interface/SiPixelPhase1Summary.h

```diff
@@ -69,10 +70,20 @@
 
        std::map<std::string,std::string> summaryPlotName_;
 
+       //The dead and innefficient roc trend plot
+       std::map<std::string,MonitorElement*>  deadROCTrends_;
```

Comment:
Hi @davidlange6 - I've changed the deadROCTrends_ and ineffROCTrends_ as you suggest, but I don't immediately see how this would work for summaryPlotName_ since both the key and value are configurable strings passed from the SiPixelPhase1Summary_cfi.py config file. I don't know where I'd start in changing that...